src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -859,12 +854,28 @@ private static bool IsViableType(LookupResult result)
 
             result.Free();
             return NamespaceOrTypeOrAliasSymbolWithAnnotations.CreateUnannotated(AreNullableAnnotationsEnabled(node.Identifier), bindingResult);
+        }
 
-            NamedTypeSymbol getNativeIntType(SyntaxNode node, bool unsigned, DiagnosticBag diagnostics)
+        /// <summary>
+        /// If the node is "nint" or "nuint", return the corresponding native integer symbol.
+        /// Otherwise return null.
+        /// </summary>
+        private NamedTypeSymbol BindNativeIntegerSymbolIfAny(IdentifierNameSyntax node, DiagnosticBag diagnostics)
+        {
+            SpecialType specialType;
+            switch (node.Identifier.ValueText)
```

Comment:
>switch (node.Identifier.ValueText) [](start = 12, length = 34)

Should we filter out escaped identifiers?

Reply:
Good catch, thanks.

---
In reply to: [392370612](https://github.com/dotnet/roslyn/pull/42360#discussion_r392370612) [](ancestors = 392370612)